django_cas_ng/views.py

```diff
@@ -221,8 +221,11 @@ def get(self, request: HttpRequest) -> HttpResponse:
 
         next_page = next_page or get_redirect_url(request)
         if settings.CAS_LOGOUT_COMPLETELY:
-            protocol = get_protocol(request)
-            host = request.get_host()
+            if hasattr(settings, 'CAS_ROOT_PROXIED_AS'):
```

Comment:
Please note that it is possible for the `settings` module to have an attr called `CAS_ROOT_PROXIED_AS` but the attraction not having a value (see https://github.com/django-cas-ng/django-cas-ng/pull/308 for more information).

So we should check for the truthiness of the setting:

```suggestion
            if hasattr(settings, 'CAS_ROOT_PROXIED_AS') and settings.CAS_ROOT_PROXIED_AS:
```

Reply:
Thank you!